src/components/uri/Visualization.tsx

```diff
@@ -1,4 +1,5 @@
-// (C) 2007-2018 GoodData Corporation
+// (C) 2007-2019 GoodData Corporation
+import { IConvertedAFM } from "@gooddata/gooddata-js/lib/DataLayer/converters/toAfmResultSpec";
```

Comment:
I'm not sure importing a type from so deep inside the library is a good idea. What if the structure of `gooddata-js` changes?

Reply:
on change, I believe code breaks regardless of this import as the toAfmResultSpec fun returns this type; object deconstruction using inferred type was in place prior this PR

Reply:
I see, OK then :)